app/javascript/components/supplemental_application/sections/LeaseInformatonInputs.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { Select, Text } from 'react-form'
+import { map, toSafeInteger } from 'lodash'
+
+import FormGrid from '~/components/molecules/FormGrid'
+import { pluck, decorateComponents } from '~/utils/utils'
+import { withNestedForm } from '~/utils/form/hoc'
+import formUtils from '~/utils/formUtils'
+import { withContext } from '../context'
+
+const Plain = ({ text }) => <div>{text}</div>
+
+const INPUTS = { 'Select': Select, 'Text': Text, 'Plain': Plain }
+
+const CustomFormGrid = decorateComponents(INPUTS, Component => {
+  return ({ label, ...rest }) => (
+    <FormGrid.Item>
+      <FormGrid.Group label={label}>
+        <Component {...rest} />
+      </FormGrid.Group>
+    </FormGrid.Item>
+  )
+})
+
+const LeaseInformatonInputs = ({ formApi, store }) => {
+  const { availableUnits } = store
+  const list = map(availableUnits, pluck('id', 'unit_number'))
```

Comment:
i think this could have a more descriptive variable name, right? availableUnitsList? or combine it with line below?

Reply:
👍